### ISSUE TITLE
`Paradox docs` link in README is dead

### ISSUE THREAD

**Issue by Friendseeker** [lightbend/paradox#630]
https://developer.lightbend.com/docs/paradox/current/ throws 404

**Comment by SethTisue** [comment]
I'm inquiring with IT staff at Akka.

**Comment by ennru** [comment]
The site was intentionally taken down.
Please browse the docs directly in the GitHub repo https://github.com/lightbend/paradox/blob/main/docs/src/main/paradox/index.md

**Comment by Friendseeker** [comment]
> The site was intentionally taken down. Please browse the docs directly in the GitHub repo https://github.com/lightbend/paradox/blob/main/docs/src/main/paradox/index.md

Thanks for reply, Enno! One problem is that Github does not seem to like that particular flavour of markdown. Directly viewing the doc via Github results in annotations such as `@@toc { depth=1 }` not rendered.

I wonder if it would be better to make a Github Page out of the existing markdown documentation?

**Comment by ennru** [comment]
Yes, it would be great to publish the site to Github pages. Please suggest the necessary changes and I'll try to get it sorted.

**Comment by Friendseeker** [comment]
> Yes, it would be great to publish the site to Github pages. Please suggest the necessary changes and I'll try to get it sorted.

I am not familiar with Markdown documentation, so Seth may need to correct me, but I guess it should be basically creating a new GitHub Repo to host the GitHub Page, run some static site generator (I guess paradox itself?) on the current markdown docs, and place the generated site directory in the created Repo.

**Comment by SethTisue** [comment]
We don't need a separate repo, just a `website` branch in this repo.

**Comment by SethTisue** [comment]
I recently did a similar transition for our Fortify docs, which are also Paradox based, and I can confirm that there really isn't anything to it besides running `sbt paradox` and then syncing the website branch with `target/paradox/site/main`

at least, there wasn't anything else I had to do there, I don't _think_ this repo is special

**Comment by SethTisue** [comment]
Maybe https://www.scala-sbt.org/sbt-site/ would be helpful here? I have no firsthand experience with it. It seems to have Paradox support: https://www.scala-sbt.org/sbt-site/generators/paradox.html

**Comment by SethTisue** [comment]
I generated the site using `sbt paradox`, made an orphan `gh-pages` branch, and pushed the generated files (from `docs/target/paradox/site/main`) to the branch

it would be nice to have some automation for when it needs updating, but I didn't do it because I'm only willing to spend a small amount of time on this

**Comment by SethTisue** [comment]
lo and behold! after I enabled Pages at https://github.com/lightbend/paradox/settings/pages , the site appeared at https://lightbend.github.io/paradox/

**Comment by SethTisue** [comment]
I've opened a separate ticket on the lack of automation: https://github.com/lightbend/paradox/issues/640
